### PR TITLE
feat: account parser

### PR DESCRIPTION
**Pull request by CalicoNino**
- Relates to #371 

This is the first step in closing #371. Since we support Eth accounts, we need to add a custom Account Parser into both the `SigningCosmWasmClient` and `SigningStargateClient`. Luckily, there is already an option for injecting custom a Account Parser into `SigningStargateClient` ([PR for reference](https://github.com/cosmos/cosmjs/pull/1102/files#diff-422398c01126667af0a2fc422f5c8e588342381f015659c5530ec46d662cb649)). 

We would also need to make a change to `SigningCosmWasmClientOptions` in `cosmjs/cosmwasm-stargate`, to also accept custom parsers https://github.com/cosmos/cosmjs/pull/1610. Until then, in a separate PR, we would expose a `signWithKeplrEVM` to mitigate developers facing #371.

## Release Notes for Version 4.5.2

- **New Features**
	- Introduced automatic labeling of newly opened issues with the "S-triage" label for better organization.
	- Added a GitHub Actions workflow to automate the addition of issues to a project board for triage and assignment.
	- Enhanced account handling by adding functions to convert Ethereum accounts to Cosmos accounts.
	- Introduced a new JSON configuration for the "cataclysm-1" blockchain, detailing network parameters.

- **Chores**
	- Updated CHANGELOG to reflect changes and version bump to 4.5.2.
	- Updated package version from 4.5.1 to 4.5.2.

### CODE REVIEW

File: src/sdk/tx/account.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * @param {Any} input - The input account information, containing the typeUrl and value.
+ * @returns {Account} Parsed account object.
+ */
+export const accountFromNibiru: AccountParser = (input: Any): Account => {
+  const { typeUrl, value } = input
+
+  if (typeUrl === "/eth.types.v1.EthAccount") {
+    const ethAccount = EthAccount.decode(value)
+    return accountFromEthAccount(ethAccount)
+  }
```
[the remark this answers is not in the document]
Comment (CalicoNino) [reply src/sdk/tx/account.ts]: Was following cosmjs standard, https://github.com/cosmos/cosmjs/blob/e819a1fc0e99a3e5320d8d6667a08d3b92e5e836/packages/stargate/src/accounts.ts#L48-L88.

If it is necessary we can wrap it in a try-catch.

[the remark this answers is not in the document]
Comment (CalicoNino) [reply src/sdk/tx/account.ts]: @cgilbe27 thoughts?

File: src/sdk/tx/account.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { BaseAccount } from "src/protojs/cosmos/auth/v1beta1/auth"
+
+/**
+ * Converts an EthAccount to a general Cosmos Account object.
+ *
+ * @param {EthAccount} ethAccount - The EthAccount object containing the account's base information.
+ * @returns {Account} The Cosmos account object.
+ */
+export const accountFromEthAccount = (baseAccount: BaseAccount): Account => {
+  const { address, pubKey, accountNumber, sequence } = baseAccount
```
Comment (cgilbe27) [inline src/sdk/tx/account.ts]: Could destructure in the function instead

const fn = ({address, pub,...}: BaseAccount) => {...}

File: src/sdk/utils/t.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "chainId": "cataclysm-1",
```
Comment (cgilbe27) [inline src/sdk/utils/t.json]: Leftover file?

File: src/sdk/tx/account.test.ts [hunk 4]
```diff
@@ -0,0 +1,105 @@
+import { accountFromEthAccount, accountFromNibiru } from "./account"
+import { EthAccount } from "src/protojs/eth/types/v1/account"
+import { Any } from "src/protojs/google/protobuf/any"
+import Long from "long"
+import * as cosmjs from "@cosmjs/stargate"
```
Comment (cgilbe27) [inline src/sdk/tx/account.test.ts:5]: Nitpick, prefer import destructure

File: src/sdk/tx/account.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (cgilbe27) [reply src/sdk/tx/account.ts]: @CalicoNino accountFromAny will throw an error if it doesn't match
image

[the remark this answers is not in the document]
Comment (cgilbe27) [reply src/sdk/tx/account.ts]: The other calls seem to be fine

File: src/sdk/utils/t.json (hunk 3 above)
Comment (CalicoNino) [reply src/sdk/utils/t.json]: not sure, will remove.

File: src/sdk/tx/account.test.ts (hunk 4 above)
Comment (CalicoNino) [reply src/sdk/tx/account.test.ts:5]: used this for spyon in L87

File: src/sdk/tx/account.ts (hunk 2 above)
Comment (CalicoNino) [reply src/sdk/tx/account.ts]: changed

**Review by cgilbe27 — APPROVED** [review]

**Pull request merged**